src/ui/Interface.js

```diff
@@ -0,0 +1,27 @@
+module.exports = function Interface (options) {
+
+    options.latId = options.latId || 'lat';
+    options.lngId = options.lngId || 'lng';
+
+    function panMapWhenInputsChange() {
+    var lat = document.getElementById(options.latId);
+    var lng = document.getElementById(options.lngId);
+
+    function panIfValue() {
+      if(lat.value && lng.value) {
+        options.panMap(lat.value, lng.value);
```

Comment:
Are we sure that this runs? Perhaps we should add a test for the panMap function directly, first, to test that assumption.